test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -25,13 +25,53 @@ Class. We use :proc:`~UnitTest.runTest` to pass the tests.
 .. code-block:: chapel
 
    use UnitTest;
+    
+   // calculates factorial
+   proc factorial(x: int): int {
+     return if x == 0 then 1 else x * factorial(x-1); 
+   }
 
    proc test1(test: Test) throws {
-     test.assertTrue(True);
+     test.skipIf(factorial(0) != 1,"Base condtion is wrong in factorial");
+     test.assertTrue(factorial(5) == 120);
    }
 
    UnitTest.runTest(test1);
 
+Another Example
+
+.. code-block:: chapel
+
+   use UnitTest;
+    
+   proc celius2fahrenheit(x) {
+     return (x * 9/5)+32;
+     // we should be returning "(x: real * 9/5)+32"
```

Comment:
Stylistically, I think these comments should come before the line they are commenting.

Reply:
Updated